persistent-storage-management/main.py

```diff
@@ -0,0 +1,86 @@
+""" 
+A little utility to view or clear files from your persistent storage.
```

Comment:
don't know if this is the best example. Just show saving a image or audio file or downloading weights from a s3 bucket if they don't exist